bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -489,13 +489,21 @@ async def _nomination_to_string(self, nomination: Nomination) -> str:
         entries_string = "\n\n".join(entries)
 
         start_date = time.discord_timestamp(nomination.inserted_at)
+
+        if nomination.thread_id:
+            thread = await self.bot.fetch_channel(nomination.thread_id)
```

Comment:
I think we need to handle if this is `None` as well (which could be the case if the thread was set but then deleted).

Could we also use the `get_or_fetch_channel` utility function (I forget where it's defined, something like `bot.utils.channel`). This would avoid an API call if the thread is cached.